### ISSUE TITLE
Fatal runtime error when using deno 1.2.0

### ISSUE THREAD

**Issue by kchair** [webview/webview_deno#40]
**OS**: macOS Catalina 10.15.5
**deno version**: 1.2.0
**v8 version**: 8.5.216
**webview version**: 0.4.2

**Code**:
```js
import { WebView } from 'https://deno.land/x/webview@0.4.2/mod.ts';

const html = `<html>yo</html>`;

const webview = new WebView({
  title: 'yo',
  url: 'data:text/html,' + html,
  width: 800,
  height: 600,
  resizable: true,
  debug: true,
  frameless: false,
});

webview.run();
```

**Output:**
```bash
% RUST_BACKTRACE=full deno run -A --unstable example.js 
Check file:///****/example.js
INFO downloading deno plugin "deno_webview" from "https://github.com/eliassjogreen/deno_webview/releases/download/0.4.2/libdeno_webview.dylib"
INFO load deno plugin "deno_webview" from local "/****/.deno_plugins/deno_webview_3d132d00f9ce9013dc21a9d6fac4877c.dylib"
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error("expected value", line: 1, column: 1)', src/lib.rs:69:36
stack backtrace:
   0:        0x106f723bf - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hdfed66b3e2e8ce60
   1:        0x106f8cd2e - core::fmt::write::h9de7b13cfaa80b03
   2:        0x106f710b7 - std::io::Write::write_fmt::h9197012b9d65a9a3
   3:        0x106f73d1a - std::panicking::default_hook::{{closure}}::h7b209cdc44c5777f
   4:        0x106f73a5c - std::panicking::default_hook::hf135bc502e6f77db
   5:        0x106f74288 - std::panicking::rust_panic_with_hook::h7967810bc33e523b
   6:        0x106f73e52 - rust_begin_unwind
   7:        0x106f9337f - core::panicking::panic_fmt::h9eea9e4965bd189c
   8:        0x106f93285 - core::result::unwrap_failed::h21e25fa0dcab039e
   9:        0x106e0b3e1 - deno_webview::op_webview_new::h9cc65cdd90605ba3
  10:        0x10311b4eb - <deno::ops::plugin::PluginInterface as deno_core::plugin_api::Interface>::register_op::{{closure}}::h0417aafddb2c6ab7
  11:        0x103af1363 - deno_core::core_isolate::CoreIsolateState::dispatch_op::h024f1e51ff678c99
  12:        0x103b00ca9 - deno_core::bindings::send::h97de4c00f705658d
  13:        0x103afdd06 - <extern "C" fn(A0) .> R as rusty_v8::support::CFnFrom<F>>::mapping::c_fn::h3b3afeb4a9ed2aba
  14:        0x103b8d65f - _ZN2v88internal25FunctionCallbackArguments4CallENS0_15CallHandlerInfoE
  15:        0x103b8cb91 - _ZN2v88internal12_GLOBAL__N_119HandleApiCallHelperILb0EEENS0_11MaybeHandleINS0_6ObjectEEEPNS0_7IsolateENS0_6HandleINS0_10HeapObjectEEESA_NS8_INS0_20FunctionTemplateInfoEEENS8_IS4_EENS0_16BuiltinArgumentsE
  16:        0x103b8c212 - _ZN2v88internalL26Builtin_Impl_HandleApiCallENS0_16BuiltinArgumentsEPNS0_7IsolateE
fatal runtime error: failed to initiate panic, error 5
zsh: abort      RUST_BACKTRACE=full deno run -A --unstable 
```

**Comment by kchair** [comment]
Looks like this issue doesn't occur on deno 1.1.3, updating title.

**Comment by mekwall** [comment]
Same thing happens on Windows 10 19041.388 64bit with deno 1.2.0.

**Comment by eliassjogreen** [comment]
Deno 1.1.3 is the latest supported version. I am currently not home and don't have access to an adequate computer so an update will have to wait unless a pr is made to support the latest version of the crate `deno_core`.

**Comment by qu4k** [comment]
This issue should be fixed with 066d5366405b0f0d50f21aa84a822fff1dd52220.
You should remove the `.deno_plugin` folder is your project directory to allow `webview_deno` to download the latest version.

**Comment by mekwall** [comment]
Confirmed fixed on Windows! Thanks @qu4k
